grunt/config/replace.js

```diff
@@ -39,10 +35,6 @@ module.exports = function (grunt, options) {
           grunt.log.writeln('WARNING: xAPI activityID has not been set');
         }
       }
-
-      // Ensure that only whitelisted attributes can be replaced.
-      courseJson = _.pick(courseJson, 'title', 'displayTitle', 'body', 'description');
-      configJson = _.pick(configJson, '_xapi', '_spoor');
```

Comment:
removed these restrictions as both files are distributed with the finished product and it'll allow more flexibility in the string replacement
https://github.com/adaptlearning/adapt_framework/issues/1798